MetasysServices/Audits/AuditFilter.cs

```diff
@@ -10,30 +10,45 @@ namespace JohnsonControls.Metasys.BasicServices
     public class AuditFilter : TimeFilter
     {
         /// <summary>
-        /// Filter by comma-separated list of origin applications.
-        /// See /enumSets/578/members for possible values.
-        /// Example: 1,2.
+        /// Holds enum value of origin applications.
         /// </summary>
-       public string OriginApplications { get; set; }
+       public IEnumerable<OriginApplicationsEnum> OriginApplicationsEnum { get; set; }
```

Comment:
```suggestion
       public OriginApplicationsEnum OriginApplications { get; set; }
```

Reply:
If we remove IEnumerable then how it will accept multiple enum values that are passed by the user.

Reply:
Flags accepts multiple values by design using pipe (|).